language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/FromClauseNodeContext.java

```diff
@@ -123,11 +128,45 @@ public void sort(BallerinaCompletionContext context,
                 TypeDescKind.STRING, TypeDescKind.ARRAY,
                 TypeDescKind.MAP, TypeDescKind.TABLE,
                 TypeDescKind.STREAM, TypeDescKind.XML);
-        
+
+        Optional<TypeSymbol> optionalTypeSymbol = Optional.empty();
+        Optional<Integer> optionalInKeyword = Optional.empty();
+        if (context.getNodeAtCursor() instanceof FromClauseNode) {
```

Comment:
Shall we use `context.getNodeAtCursor().kind() == SyntaxKind.FROM_CLAUSE` instead of `instanceof` check?

Reply:
yes, its better